access/processor.go

```diff
@@ -174,9 +174,17 @@ func (a *atp) keyFunc(token *jwt.Token) (interface{}, error) {
 		return nil, errors.New("kid not written in header")
 	}
 
-	key := a.jwkp(keyID.(string))
+	jwkSetURL, ok := token.Header["jku"]
```

Comment:
定義済みのconstを使ってはどうでしょうか（`kid`も同様に）
```suggestion
	jwkSetURL, ok := token.Header[jws.JWKSetURLKey]
```

Reply:
ライブラリ側でキーが定義されているんですね！ありがとうございます、修正しました。 49a6372